snippets/language-html.cson

```diff
@@ -121,7 +121,7 @@
     'body': '<footer>$1</footer>$0'
   'Form':
     'prefix': 'form'
-    'body': '<form id="${1:form_id}" action="${2:index.html}" method="${3:post}">\n\t$4\n</form>'
+    'body': '<form class="${1:form_class}" action="${2:index.html}" method="${3:post}">\n\t$4\n</form>'
```

Comment:
I'd say just a cursor for class since it is pretty well understood and any default will almost never be used (vs. something like method which the template helps you with since it is a sensible default)